src/components/Checkbox/Checkbox.stories.tsx

```diff
@@ -36,47 +34,55 @@ export default {
 
 type Args = React.ComponentProps<typeof Checkbox>;
 
-/**
- * Controlled example to make checked stories interactive.
- */
-function CheckboxExample(args: Args) {
-  const [checked, setChecked] = React.useState<
-    boolean | 'indeterminate' | undefined
-  >(args.checked);
-  const handleChange = () => {
-    setChecked(!checked);
-  };
-
-  return <Checkbox checked={checked} onChange={handleChange} {...args} />;
-}
-
-export const Default: StoryObj<Args> = {
-  render: (args) => <CheckboxExample {...args} />,
-};
+export const Default: StoryObj<Args> = {};
 
 export const Checked: StoryObj<Args> = {
-  render: (args) => <CheckboxExample {...args} checked />,
+  ...Default,
+  args: {
+    defaultChecked: true,
+  },
 };
 
 export const Medium: StoryObj<Args> = {
-  render: (args) => <CheckboxExample {...args} size="md" />,
+  ...Default,
+  args: {
+    size: 'md',
+  },
 };
 
 export const MediumChecked: StoryObj<Args> = {
-  render: (args) => <CheckboxExample {...args} checked size="md" />,
+  ...Medium,
+  args: {
+    ...Checked.args,
+    ...Medium.args,
+  },
+};
+
+export const Large: StoryObj<Args> = {
+  ...Default,
+  args: {
+    size: 'lg',
+  },
+};
+
+export const LargeChecked: StoryObj<Args> = {
+  ...Large,
```

Comment:
Not sure what spreading all these Stories are doing

Reply:
Clones the Large story.

Yeah there's not actually any properties being brought over currently. But makes it clear that this story is based on Large.